README.md

```diff
@@ -7,8 +7,13 @@
 [![Coveralls Coverage](https://coveralls.io/repos/github/SudoKuru/Sudokuru/badge.svg?branch=main)](https://coveralls.io/github/SudoKuru/Sudokuru?branch=main)
 [![Codecov Coverage](https://codecov.io/gh/SudoKuru/Sudokuru/graph/badge.svg?token=SISS85TX20)](https://codecov.io/gh/SudoKuru/Sudokuru)
 
+[![Common Changelog](https://common-changelog.org/badge.svg)](https://common-changelog.org)
```

Comment:
Can you update the git PR checklist to include updating the changelog (using [common changelog format](https://common-changelog.org/)) if needed?